### ISSUE TITLE
MPE Stuck Pitch Bend

### ISSUE THREAD

**Issue by gustebeast** [asb2m10/dexed#459]
I've run into a pretty glaring issue when using MPE where the pitch bend gets stuck and I end up having everything unexpectedly transposed. Here's a video demonstrating it. At the start I show the dexed version with the unexpected transposition, and at the end I run the MIDI through a different MPE synth (pigments) to show what it's supposed to sound like.

https://www.youtube.com/watch?v=zwE6fF-hVtg

Here's the MIDI file for that demo:
https://drive.google.com/file/d/17yqPsqzC3aMY7jiJm-KTC7q-xW38Ndzs/view?usp=sharing

Note that this is using a bend range of 48, not the default 24.

It's hard for me to know what exactly is going wrong here, but my suspicion is that it has something to do with triggering the same note value on multiple channels. I made a second MIDI file focused on that specifically, and it more clearly demonstrates an issue, although I'm not sure if it's the same issue as above.

https://www.youtube.com/watch?v=e0eh4tnsK7I

And the MIDI file:
https://drive.google.com/file/d/1ZX39OvuJa5LD0CHKbgulW__Pp_MO6Ta2/view?usp=sharing

If I play two copies of a note in unison across two channels, then bend one down, it works perfectly fine, so dexed does seem capable of tracking two copies of the same note on separate channels with separate bends. It only becomes a problem once you bend both notes up or down a couple semitones, as demonstrated in the video above.

**Comment by gustebeast** [comment]
cc @baconpaul since you initially added MPE support and probably have helpful context here

**Comment by baconpaul** [comment]
Yeah I’ll look this week thanks for the excellent report

**Comment by gustebeast** [comment]
@baconpaul I figured out what was wrong. I had manually rechanalized my MPE to use channels 1-10, but channel 1 is supposed to be reserved for non-MPE instruments. The MIDI demo above uses channel 1 alongside the other channels, hence the issue.

I updated my rechannalizer to use channels 2-11 and now the same note sequence plays back flawlessly. I'll close this since I don't think dexed should have to accommodate usage outside of the midi spec.
